apps/web-giddh/src/app/shared/header/components/account-update-new-details/account-update-new-details.module.ts

```
@@ -13,6 +13,7 @@ import { MatButtonModule } from "@angular/material/button";
 import { MatRadioModule } from "@angular/material/radio";
 import { MatTabsModule } from "@angular/material/tabs";
 import { NewConfirmModalModule } from "apps/web-giddh/src/app/theme/new-confirm-modal";
+import { SalesPersonService } from "../../../sales-person/utility/sales-person.service";
 import { MatTooltipModule } from "@angular/material/tooltip";
 import { SalesPersonService } from "../../../sales-person/utility/sales-person.service";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'SalesPersonService'. Consider to delete it or rename it.

'SalesPersonService' is defined here:


(lint/suspicious/noRedeclare)